arm/Microsoft.DataFactory/factories/managedVirtualNetwork/readme.md

```diff
@@ -0,0 +1,27 @@
+# DataFactoryManagedVirtualNetwork `[Microsoft.DataFactory/factories/managedVirtualNetworks]`
```

Comment:
Same as before, please update to a more meaningful name & add a description

Reply:
Updated!